### DISCUSSION TITLE
Stubbing of HTML response

### DISCUSSION THREAD

**Discussion by mayankCloud** [specmatic/specmatic#1012] [Ask For Help]
Is there a way to stub out APIs that respond back in HTML

For example below API responds back a status of 401, but the response body is in HTML format.
When I am doing specmatic proxy i get the error - XML not supported yet.

<img width="1634" alt="Screenshot 2024-03-04 at 6 55 53 PM" src="https://github.com/znsio/specmatic/assets/63167098/dcf919df-1101-4856-a441-9310184b9743">

**Comment by harikrishnan83** [comment]
@mayankCloud thanks for sharing the details. We will take a look.
